static/package.json

```diff
@@ -45,5 +46,5 @@
       "last 1 safari version"
     ]
   },
-  "proxy": "http://localhost:5000"
+  "proxy": "https://localhost"
```

Comment:
You are also assuming that nothing else is already running an HTTPS server on the system that is running or being used to develop the daemon.

Reply:
within the container, nothing else should be running on that port. The container can be mapped to any port on the host.

Reply:
This line is never used in the container, only when the webpack dev server is running. Using 443 means that anyone trying to contribute to this project would need to turn off existing any existing servers on that port.